reference/api/state.md

```diff
@@ -42,13 +47,14 @@ This endpoint lets you save an array of state objects.
 
 ### HTTP Request
 
-`POST http://localhost:<daprPort>/v1.0/state`
+`POST http://localhost:<daprPort>/v1.0/state/<storename>`
 
 #### URL Parameters
 
 Parameter | Description
 --------- | -----------
 daprPort | the Dapr port
+storename | ```metadata.name``` field which is the name of the state store
```

Comment:
metadata.name is not self explanatory here. WE should clarify that its the name user has configured in the component.yaml for the state store.